HelloWorld/.vscode/extensions.json

```diff
@@ -3,6 +3,7 @@
     "salesforce.salesforcedx-vscode",
     "redhat.vscode-xml",
     "dbaeumer.vscode-eslint",
-    "esbenp.prettier-vscode"
+    "esbenp.prettier-vscode",
+    "financialforce.lana"
```

Comment:
Do we need `financialforce`?

Reply:
These are all auto-created when you use VSCode SFDX plugin to create a new project. I left them all as is.